site/content/en/docs/tasks/_index.md

```diff
@@ -50,3 +50,10 @@ A _platform developer_ integrates Kueue with other software and/or contributes t
 As a platform developer, you can learn how to:
 - [Integrate a custom Job with Kueue](/docs/tasks/integrate_a_custom_job).
 - [Enable pprof endpoints](/docs/tasks/enabling_pprof_endpoints).
+
+## Troubleshooting
+
+Sometimes things go wrong. The following guides can help you understand the state
+of the system.
+
+- [Troubleshooting Jobs](troubleshooting/troubleshooting_jobs)
```

Comment:
On the deploy preview this link does not work correctly

Reply:
I'm not sure why it's not showing yet. I'm checking

Reply:
ahhh, it was because I was using a future date :)